### ISSUE TITLE
when I set trust degrees parameters in the librec.conf , I confront a problem. 

### ISSUE THREAD

**Issue by Jakye** [guoguibing/librec#9]
When I set the trust degrees parameters in the librec.conf to do experiments , I   encountered  a problem which is showing as follow:
2015-03-25 17:04:16,065 [INFO] Training: F:\dataset\Epinions\ratings.txt, ratio: 0.8, trust-degree [1, 5]
2015-03-25 17:04:19,125 [DEBUG] Dataset: {Users, Items, Ratings} = {40163, 139738, 664824}, Scale = {1.0, 2.0, 3.0, 4.0, 5.0}
2015-03-25 17:04:19,736 [DEBUG] training amount: 531982, test amount: 132842
2015-03-25 17:04:19,736 [DEBUG] Social dataset: F:\dataset\Epinions\trust.txt
2015-03-25 17:04:21,216 [DEBUG] Dataset: {Users, Users, Links} = {49289, 49289, 487183}, Scale = {1.0}
2015-03-25 17:04:22,173 [DEBUG] TrustSVD: 0.01,-1.0,0.9,0.9,0.9,0.5,10,100,true
2015-03-25 17:04:45,640 [DEBUG] TrustSVD iter 1: errs = 345709.44, delta_errs = -345709.44, loss = 885255.7, delta_loss = -885255.7, learn_rate = 0.01
2015-03-25 17:05:08,851 [DEBUG] TrustSVD iter 2: errs = 320324.3, delta_errs = 25385.13, loss = 5924772.0, delta_loss = -5039516.5, learn_rate = 0.01
2015-03-25 17:05:32,294 [DEBUG] TrustSVD iter 3: errs = NaN, delta_errs = NaN, loss = NaN, delta_loss = NaN, learn_rate = 0.005
2015-03-25 17:05:32,294 [ERROR] Loss = NaN or Infinity: current settings cannot train the recommender! Try other settings instead!

The main configuration parameters in the librec.conf  which I has changed is showing as follow:
dataset.training.wins=F:\\dataset\\Epinions\\ratings.txt
dataset.social.wins=F:\\dataset\\Epinions\\trust.txt
min.trust.degree=1
max.trust.degree=5
rating.pred.view=trust-degree
val.reg.user=0.9
val.reg.item=0.9
val.reg.bias=0.9
val.reg.social=0.5
num.factors=10
recommender=TrustSVD

I have encountered  the NaN or Infinity problem When I did experiments. Finally, I resolved it by sorted the records in ratings.txt and  trust.txt of Epinions. Why I got the problem again ? Is the parameters wrong in  in the librec.conf?

**Comment by guoguibing** [comment]
For trust-degree, you may need to try some other settings.
